[Community Class] Jedi Kindred Duo

airchromemb2 opened the pull request:
Two jedi classes specifically meant to be played in tandem, weak on their own but strong together. Both of these classes utilize lights beacon and protect/absorb with a 200% longer skilltimer to prevent constantly having their buffs on, they do have 120hp and fp though to give them a small edge because of their otherwise lackluster kit.
I removed the Barriss/Luminara models/sabers from Jedi Healer/Traditionalist respectively as the duo classes utilize them.

Notable Overview:

- Jedi Kindred (Absorb) has pull 2, blaster defense 2, and 10% more BP whilst the rest of their kit is level 1.
- Jedi Kindred (Protect) has push 2, saber defense 2, and 10% more AP whilst the rest of their kit is level 1.

airchromemb2 committed:
[Community Class] Jedi Kindred Duo

Two jedi classes specifically meant to be played in tandem, weak on their own but strong together. Both of these classes utilize lights beacon and protect/absorb with a 200% longer skilltimer to prevent constantly having their buffs on, they do have 120hp and fp though to give them a small edge because of their otherwise lackluster kit.
I removed the Barriss/Luminara models from jedi healer/traditionalist respectively as these classes utilize them.

Notable Overview:

- Jedi Kindred (Absorb) has pull 2, blaster defense 2, and 10% more BP whilst the rest of their kit is level 1.
- Jedi Kindred (Protect) has push 2, saber defense 2, and 10% more AP whilst the rest of their kit is level 1.

airchromemb2 committed:
Removing Barriss Model/Saber from Jedi Healer

Removed Barriss Offee model and saber from Jedi Healer as Jedi Kindred uses them.

airchromemb2 committed:
Removing Luminara Model/Saber from Jedi Traditionalist

Removed Luminara Unduli model and saber from Jedi Traditionalist as Jedi Kindred uses them.

airchromemb2 commented:
ignore the pr graveyard i have a brain worm and it is dying of hungry

nomad-marshall commented:
remember to add your classes to the team file .mbtc

check your pull requests!!

nomad-marshall commented:
> the absorb variant is the one that possesses pull 2 and push 2 is on protect since pull is typically better against gunners, giving it to the jedi with red/yellow/protect i think would create a power imbalance favoring the protect variant having increased AP, protect, pull. i debated giving protect to the jedi that currently has absorb but since the AP variant with SD2 will most likely be front lining to duel i think it worked best on them but im open to that change as that decision has pros for both arguments.

okay i think i'm understanding a little more what you're doing. it's really interesting. i like this dichotomy of effectiveness.  i'll reaffirm my suggestion for increasing their respective specialized force power to 3, and if i knew more about the levels of absorb/protect (i think they just decrease the cooldown) id recommend that as well. what do you think about adding level 1 of drain and lightning to each? too far out of the scope/acceptable lore? regardless, this is an awesome concept and i'd love to see it in the game

airchromemb2 commented:
im pretty certain higher levels of protect/absorb only decrease the fp cost and the cooldown remains the same, which alongside the increase in level to pull/push of each class it might be worth considering upping their level so they have better fp sustain, perhaps the protect variant could have level 2 since they have bd1. 

as for doing drain/lightning id prefer to leave those to potential sith duo classes in the future if the classes get added and are well received but i think its a great idea, it would be really cool to see the juxtaposition and thus the fights they could have with one another. 

your idea did give me inspiration though, what if the protect variant had blind and the absorb variant had deadly sight? since absorb in theory will be a more supporting role and protect having blind would promote chaos and confusion, especially since blind is harder to miss at closer range/frontline since your target is larger and you can capitalize off of the blind with your saber if you are near them.

also, i think giving both of them deflect two instead of one could be a great addition as well, speaking of additions im going to be adding them to the mbtc now lol

airchromemb2 committed:
added jedi kindred classes to .mbtc

added jedi kindred classes to the team config for legends

airchromemb2 committed:
giving kindred deadly sight and restructuring class special binds

made it so cs1 is saber throw, cs2 slap, cs3 absorb and cs4 deadly sight

airchromemb2 committed:
gave kindred protect new force powers and restructured class special binds

upped push and jump to level 3, added blind level 1, restructured class specials to go saberthrow, slap, protect, blind on cs1,2,3,4 respectively

airchromemb2 committed:
finishing touches on kindred absorb class

forgot to add some things on first pass, gave kindred absorb pull and jump level 3, updated class desc

airchromemb2 committed:
kprotect had empty spaces in desc, fixed

a couple lines in desc were blank, removed them

airchromemb2 committed:
kabsorb had empty spaces in desc, fixed

same story, a few random blank lines after desc, they weren't effecting anything but meh
